lib/src/over_react_test/common_component_util.dart

```diff
@@ -535,10 +535,12 @@ void testRequiredProps(BuilderOnlyUiFactory factory, dynamic childrenFactory())
 
     for (var consumedProp in consumedProps) {
       for (var prop in consumedProp.props) {
-        if (prop.isNullable) {
-          nullableProps.add(prop.key);
-        } else if (prop.isRequired) {
-          requiredProps.add(prop.key);
+        if (!prop.isLate) {
```

Comment:
These changes look good, but would you mind adding a test that verifies the `late` props issue we were seeing longer occurs?

Here's the test suite for commonComponentTests: https://github.com/Workiva/over_react_test/blob/v3_wip/test/over_react_test/common_component_util_test.dart